toppra/dracula/three_segment_spline.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function
```

Comment:
If this is running in python 3, you don't need this line

Reply:
removed